service/src/main/kotlin/io/provenance/explorer/domain/models/explorer/ExplorerModels.kt

```diff
@@ -15,6 +16,14 @@ data class BlockSummary(
     val txNum: Int
 )
 
+data class BlockProposer(
+    var blockHeight: Int,
+    var proposerOperatorAddress: String,
+    var blockTimestamp: DateTime,
+    var minGasFee: Double? = null,
```

Comment:
gas fees are typed double? should that be long / biginteger?

Reply:
This is due to the OG min gas fee amount, which was 0.025 nhash